ocaml/parsing/builtin_attributes.ml

```diff
@@ -523,9 +523,11 @@ let zero_alloc_attribute (attr : Parsetree.attribute)  =
   parse_attribute_with_ident_payload attr
     ~name:"zero_alloc" ~f:(function
       | "check" -> Clflags.zero_alloc_check := true
+      | "assert-all" | "assert" | "assert_all" | "all" ->
+        Clflags.zero_alloc_check_assert_all := true
       | _ ->
         warn_payload attr.attr_loc attr.attr_name.txt
-          "Only 'check' is supported")
+          "Only 'check' and 'all' are supported")
```

Comment:
oh, good catch, thanks! Fixed. 

I wanted to use "assert all" payload as a synonym for "all", but it might be confusing with "assert" keyword, as @xclerc pointed out. Looks like I removed it from `translattribute.ml` but forgot `builtin_attributes.ml`.